src/responder/common/responder_common.c

```diff
@@ -1194,6 +1194,17 @@ int sss_process_init(TALLOC_CTX *mem_ctx,
         }
     }
 
+    ret = confdb_get_string_as_list(rctx->cdb, rctx,
+                                    CONFDB_MONITOR_CONF_ENTRY,
+                                    CONFDB_MONITOR_LOOKUP_ORDER,
+                                    &rctx->lookup_order_list);
+    if (ret != EOK) {
+        DEBUG(SSSDBG_MINOR_FAILURE,
```

Comment:
Nice catch! I didn't realize the big warning in the confdb_get_string_as_list() code that says
```
 586 /* WARNING: Unlike other similar functions, this one does NOT take a default,
 587  * and returns ENOENT if the attribute was not found ! */

```